### PR TITLE
[SVL-001] add 2 test scenario

### PR DESCRIPTION
**Pull request by oksanaua**

### CODE REVIEW

File: src/test/java/actions/WebShop_Actions.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    public void addToCartSecondJewelry(){
+        webshop_elements.secondJewelry.click();
+    }
+
+    public int getTheQuantityOfTheProduct(){
+        return webshop_elements.cartItemRows.size();
+    }
+
```
Comment (serhiiluhovyi) [inline src/test/java/actions/WebShop_Actions.java:48]: For what you need this empty lines?

File: src/test/java/steps/WebShop_Steps.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    @And("I click on Jewelry Category")
+    public void iClickOnJewelryCategory() {
+        webshop_actions.clickOnJewelryCategory();
+    }
+
+    @And("I add to cart the second jewelry")
+    public void iAddToCartTheSecondJewelry() {
+        webshop_actions.addToCartSecondJewelry();
+
```
Comment (serhiiluhovyi) [inline src/test/java/steps/WebShop_Steps.java:70]: Please remove also here line

File: src/test/java/steps/WebShop_Steps.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    @And("I check the quantity {int} of the product")
+    public void iCheckTheQuantityOfTheProduct(int count) {
+        int actualCount = webshop_actions.getTheQuantityOfTheProduct();
+
+        if (actualCount !=count) {
+            fail("Incorrect quantity" + count + " results shown");
```
Comment (serhiiluhovyi) [inline src/test/java/steps/WebShop_Steps.java:78]: Please provide info about the actual count too

Comment (oksanaua) [reply src/test/java/steps/WebShop_Steps.java]: done

File: src/test/java/steps/WebShop_Steps.java (hunk 2 above)
Comment (oksanaua) [reply src/test/java/steps/WebShop_Steps.java]: done

File: src/test/java/actions/WebShop_Actions.java (hunk 1 above)
Comment (oksanaua) [reply src/test/java/actions/WebShop_Actions.java]: done

**Review by serhiiluhovyi — APPROVED** [review]

**Pull request merged**